src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -213,8 +211,8 @@ class GcsServer {
   /// Print the asio event loop stats for debugging.
   void PrintAsioStats();
 
-  /// Get or connect to a redis server
-  std::shared_ptr<RedisClient> CreateRedisClient(instrumented_io_context &io_service);
+  std::unique_ptr<RedisStoreClient> CreateRedisStoreClient(
```

Comment:
So a rabbit hole I've been down lately is that we make liberal use of instrumented_io_context in a lot of places where we don't seem to intend to use the instrumentation.  Will we always use the instrumentation when creating a redis client (though I realize it was doing so before).  If not we can just use the raw boost one.